src/resources/closure.lib.d.ts

```diff
@@ -5,3 +5,7 @@ declare namespace ಠ_ಠ.clutz_internal {
   type GlobalError = Error;
   var GlobalError: ErrorConstructor;
 }
+
+// Closures' goog namespace is accessible as global symbol without need for
```

Comment:
Give me a break :) You lose _the_ ability to type after bisecting `base.js` for the magic incantation that keeps `goog` in the provide table.